### ISSUE TITLE
Processing a Class tsv adn getting Exception in thread "main" java.util.NoSuchElementException: 

### ISSUE THREAD

**Issue by chris-day** [henrietteharmse/uml2semantics#11]
[class.zip](https://github.com/user-attachments/files/16868227/class.zip)
Hi Henriette -

I've found an issue when try to convert my attached class.tsv file

java -jar ./target/scala-3.2.2/uml2semantics.jar -c "examples/iso20022/class.tsv" --ontology iso20022.owl  

`10:49:51.615 [main] INFO  org.uml2semantics.Main$package -- Start
10:49:51.622 [main] DEBUG org.uml2semantics.Main$package -- arguments = List(-c, examples/iso20022/class.tsv, --ontology, iso20022.owl) at line 80 of Main.scala
10:49:51.882 [main] DEBUG org.uml2semantics.model.Curie -- s=AccountContract at line 119 of PrefixNamespace.scala
Exception in thread "main" java.util.NoSuchElementException: None.get
        at scala.None$.get(Option.scala:627)
        at scala.None$.get(Option.scala:626)
        at org.uml2semantics.model.UMLClassParentNamedElements$.$anonfun$2(UMLClassDiagram.scala:245)
        at scala.collection.StrictOptimizedIterableOps.map(StrictOptimizedIterableOps.scala:100)
        at scala.collection.StrictOptimizedIterableOps.map$(StrictOptimizedIterableOps.scala:87)
        at scala.collection.immutable.Set$Set1.map(Set.scala:156)
        at org.uml2semantics.model.UMLClassParentNamedElements$.fromSetOfStrings(UMLClassDiagram.scala:245)
        at org.uml2semantics.reader.TSVReader$package$.parseClasses$$anonfun$1(TSVReader.scala:56)
        at scala.collection.immutable.List.foreach(List.scala:333)
        at org.uml2semantics.reader.TSVReader$package$.parseClasses(TSVReader.scala:60)
        at org.uml2semantics.reader.TSVReader$package$.parseUMLClassDiagram(TSVReader.scala:195)
        at org.uml2semantics.Main$package$.uml2owl(Main.scala:85)
        at org.uml2semantics.uml2owl.main(Main.scala:77)`

**Comment by henrietteharmse** [comment]
Hi Chris,
When specify ParentNames, the parent name must already be defined as a class. If I use classes.tsv, I get the same error as you did:
![Highlight](https://github.com/user-attachments/assets/0d1a0581-c777-4535-a4eb-9add6d2dc812)

It actually tells you which class it could not find. That is circled in the screenshot.

I have updated your TSV file to add this, but many of the parent names have not been defined as classes. I did a few.

[Iso20022 - classes.zip](https://github.com/user-attachments/files/16886879/Iso20022.-.classes.zip)
